purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -1536,46 +1553,6 @@ class PurchasesTest {
         assertThat(callCount).isEqualTo(0)
     }
 
-    @Test
-    fun `sends cached purchaser info to getter`() {
-        every {
-            mockBackend.getCustomerInfo(any(), any(), captureLambda(), any())
-        } answers {
-            // Timeout
-        }
-
-        var receivedInfo: CustomerInfo? = null
-        purchases.getCustomerInfoWith(onSuccess = {
-            receivedInfo = it
-        }, onError = {
-            fail("supposed to be successful")
-        })
-
-        assertThat(receivedInfo).isEqualTo(mockInfo)
-    }
-
-    @Test
-    fun `given no cached purchaser info, backend is called again`() {
-        every {
-            mockCache.getCachedCustomerInfo(any())
-        } returns null
-        every {
-            mockBackend.getCustomerInfo(any(), any(), captureLambda(), any())
-        } answers {
-            // Timeout
-        }
-
-        var receivedInfo: CustomerInfo? = null
-        purchases.getCustomerInfoWith(onSuccess = {
-            receivedInfo = it
-        }, onError = {
-            fail("supposed to be successful")
-        })
-
-        assertThat(receivedInfo).isEqualTo(null)
-        verify(exactly = 1) { mockBackend.getCustomerInfo(any(), any(), any(), any()) }
-    }
```

Comment:
These are covered by other tests, and the fact that we have moved this logic out of here.